src/Tizen.Uix.Stt/Tizen.Uix.Stt/SttClient.cs

```diff
@@ -389,7 +393,11 @@ public SttClient()
                             if (data != null && msg != null)
                             {
                                 RecognitionResultEventArgs args = new RecognitionResultEventArgs(e, data, dataCount, Marshal.PtrToStringAnsi(msg));
-                                _recognitionResult?.Invoke(this, args);
+
+                                lock (_recognitionResultLock)
+                                {
+                                    _recognitionResult?.Invoke(this, args);
+                                }
```

Comment:
No need lock here, `_recognitionResult?.`  it guarantee thread safe
```suggestion
                                _recognitionResult?.Invoke(this, args);
```